qdrant-landing/content/documentation/concepts/points.md

```diff
@@ -1632,12 +1631,97 @@ All resulting points are sorted by ID. To query the next page it is necessary to
+
+<aside role="status">Without an appropriate index, custom ordering would be a very expensive operation. For this reason, in Qdrant, it is a requirement to have an index capable of <a href=/documentation/concepts/indexing/#payload-index target="_blank">Range filtering conditions</a> on the field you want to `order_by`. Once the index is set up, these ordering operations can be really cheap.</aside>
+
+```http
+POST /collections/{collection_name}/points/scroll
+{
+    "limit": 15,
+    "order_by": "timestamp", // <-- this!
+}
+```
 
 ```python
+client.scroll(
+    collection_name="{collection_name}",
+    limit=15,
+    order_by="timestamp", # <-- this!
+)
 ```
- -->
+
+```typescript
+client.scroll("{collection_name}", {
+  limit: 15,
+  order_by: "timestamp", // <-- this!
+});
+```
+
+```rust
+use qdrant_client::qdrant::{Condition, Filter, ScrollPoints, OrderBy};
+
+client
+    .scroll(&ScrollPoints {
+        collection_name: "{collection_name}".to_string(),
+        limit: Some(15),
+        order_by: Some(OrderBy {
+            key: "timestamp".to_string(),  // <-- this!
+            ..Default::default(),
+        }),
+        ..Default::default()
+    })
+    .await?;
+```
+
+The `order_by` `key` parameter specifies the payload key to order the results by, but there are also other fields that can be set to control the ordering, like `direction` and `start_from`:
+
+```http
+"order_by": {
+    "key": "timestamp",
+    "direction": "desc" // default is "asc"
+    "start_from": 123, // start from this value
+}
+```
+
+```python
+order_by=models.OrderBy(
+    key="timestamp",
+    direction="desc",  # default is "asc"
+    start_from=123,  # start from this value
+)
+```
+
+```typescript
+order_by: {
+    key: "timestamp",
+    direction: "desc", // default is "asc"
+    start_from: 123, // start from this value
+}
+```
+
+```rust
+order_by: Some(OrderBy {
+    key: "timestamp".to_string(),
+    direction: Some(Direction::Desc) // default is Direction::Asc
+    start_from: Some(123),
+})
+```
+
+Be mindful that, for array-like payloads, each point can appear as many times as the number of elements in the array. For example, if you have a point payload with a `timestamp` key, and the value for the key is an array of 3 elements, the same point will appear 3 times in the results, one for each timestamp.
+
+<aside role="alert">Pagination is disabled when using the `order_by` parameter. </aside>
+
+As you may expect, pagination does not work exactly the same way than when the order is an ID, which is unique. Right now, pagination is not explicitly enabled when ordering by a payload key, so the `next_page_offset` field will not be present in the response. However, there is a trick you can do to achieve pagination:
```

Comment:
```suggestion
Pagination works differently when  `order_by` is an ID. Since pagination is not explicitly enabled, you do not see the `next_page_offset` field in the response. However, you can use the following steps to set up pagination:
```